### PR TITLE
Add Ubiqod by Skiply v2 connector

### PR DESCRIPTION
**Pull request by Sylvain-Chu**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

This pull request adds a new Ubiqod By Skiply v2 connector. With version 1 being deprecated, we are creating another one with a brand new API and lots of new features.

### CODE REVIEW

**Comment by Sylvain-Chu** [comment]
@microsoft-github-policy-service agree company="Skiply"

File: certified-connectors/Ubiqod by Skiply (v2)/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3049 @@
...
+            "description": "403",
+            "schema": {
+              "$ref": "#/definitions/HttpError"
+            }
+          }
+        }
+      },
+      "post": {
+        "summary": "Create a QR Code Tracker (QOD)",
+        "description": "Create a new QR Code travker (QOD)",
```
Comment (vmanoharas) [inline certified-connectors/Ubiqod by Skiply (v2)/apiDefinition.swagger.json]: It seems this is a typo, can you please correct this one "travker"

File: certified-connectors/Ubiqod by Skiply (v2)/Readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Ubiqod by Skiply
```
Comment (vmanoharas) [inline certified-connectors/Ubiqod by Skiply (v2)/Readme.md]: Can you please update readme.md file as per this template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Hello @sylvain-Chu


Your merge commit has an issue, it has not merged properly, please refer the Swagger validation checker errors, and apiProperties.json file cannot be picked by validator. Please recommit or close this PR and open a new PR. Please don't merge it until the review is completed.

**Comment by vmanoharas** [comment]
Hello @Sylvain-Chu

Your merge commit has an issue, it has not merged properly, please refer the Swagger validation checker errors, and apiProperties.json file cannot be picked by validator. Please recommit or close this PR and open a new PR. Please don't merge it until the review is completed.

**Comment by Sylvain-Chu** [comment]
> Hello @Sylvain-Chu
> 
> Your merge commit has an issue, it has not merged properly, please refer the Swagger validation checker errors, and apiProperties.json file cannot be picked by validator. Please recommit or close this PR and open a new PR. Please don't merge it until the review is completed.

Hello @vmanoharas,

Thank you for the feedback. I have checked the Swagger validator on my side with paconn and it only returns warnings, no errors. Could you please provide more details on the issues ?

Thank you for your assistance.

**Comment by vmanoharas** [comment]
Hello @Sylvain-Chu, 

Thank you for validating again, however you have merged the branch before the review, kindly recommit the PR and don't merge it until the review. 

image

**Comment by Sylvain-Chu** [comment]
A new pull request has been created to address the issues in this one. Please refer to [#3509](https://github.com/microsoft/PowerPlatformConnectors/pull/3509)

**Pull request closed without merging**